code/game/machinery/biogenerator.dm

```diff
@@ -639,4 +639,32 @@
 			man_rating += P.rating
 
 	build_eff = man_rating
-	eat_eff = bin_rating
\ No newline at end of file
+	eat_eff = bin_rating
+
+/obj/machinery/biogenerator/small
```

Comment:
This inherits component_types, meaning it'll *probably* be able to be dismantled and reassembled to get a normal one.